q2_fondue/sequences.py

```diff
@@ -41,12 +42,17 @@
 
 
 def _run_cmd_fasterq(
-        acc: str, output_dir: str, threads: int):
+        acc: str, output_dir: str, threads: int, key_file: str):
     """Runs fasterq-dump command on a single accession."""
-    cmd_prefetch = ['prefetch', '-X', 'u', '-O', acc, acc]
+    if key_file != '':
+        key_params = ['--ngc', key_file]
+    else:
+        key_params = []
+
+    cmd_prefetch = ['prefetch', '-X', 'u', '-O', acc] + key_params + [acc]
```

Comment:
A more Pythonic, cleaner way to do it would be:
```suggestion
    cmd_prefetch = ['prefetch', '-X', 'u', '-O', acc, *key_params, acc]
```